content/hi/docs/tutorials/kubernetes-basics/deploy-app/deploy-interactive.html

```diff
@@ -0,0 +1,51 @@
+---
+title: इंटरएक्टिव ट्यूटोरियल - एक ऐप की डेप्लॉयमेंट
+weight: 20
+---
+
+<!DOCTYPE html>
+
+<html lang="en">
+
+<body>
+
+<link href="/docs/tutorials/kubernetes-basics/public/css/styles.css" rel="stylesheet">
+<link href="/docs/tutorials/kubernetes-basics/public/css/overrides.css" rel="stylesheet">
+<script src="https://katacoda.com/embed.js"></script>
+
+<div class="layout" id="top">
+
+    <main class="content katacoda-content">
+
+        <div class="row">
+            <div class="col-md-12">
+                <p>
+                    पॉड कुबेरनेट्स एप्लिकेशन की मूल निष्पादन इकाई है। प्रत्येक पॉड आपके क्लस्टर पर चल रहे कार्यभार के एक हिस्से का प्रतिनिधित्व करता है। <a href="/docs/concepts/workloads/pods/">पॉड्स के बारे में अधिक जानें</a>।
+                </p>
+            </div>
+        </div>
+
+        <br>
+        <div class="katacoda">
+            <div class="katacoda__alert">
+                टर्मिनल का उपयोग करने के लिए, कृपया डेस्कटॉप या टैबलेट मोड का उपयोग करें
+            </div>
+
+            <div class="katacoda__box" id="inline-terminal-1"  data-katacoda-id="kubernetes-bootcamp/7" data-katacoda-color="326de6" data-katacoda-secondary="273d6d" data-katacoda-hideintro="false" data-katacoda-prompt="Kubernetes Bootcamp Terminal" style="height: 600px;">
+            </div>
+
+        </div>
+        <div class="row">
+            <div class="col-md-12">
+            	<a class="btn btn-lg btn-success" href="/docs/tutorials/kubernetes-basics/create-cluster/cluster-intro/" role="button"> &lt; मॉड्यूल 1 पर लौटें<span class=""></span></a>
+            	<a class="btn btn-lg btn-success" href="/docs/tutorials/kubernetes-basics/" role="button">होम<span class=""></span></a>
+                <a class="btn btn-lg btn-success" href="/docs/tutorials/kubernetes-basics/explore/explore-intro/" role="button">मॉड्यूल 3 जारी रखें &gt;<span class=""></span></a>
```

Comment:
```suggestion
                <a class="btn btn-lg btn-success" href="/docs/tutorials/kubernetes-basics/explore/explore-intro/" role="button">मॉड्यूल 3 पर जाए &gt;<span class=""></span></a>
```

I know it will change the meaning of the sentence a bit. My goal is to keep the wording consistent.